src/mongo/standalong.rs

```diff
@@ -0,0 +1,105 @@
+use testcontainers::{core::WaitFor, Image};
+
+use super::{NAME, TAG};
+
+#[derive(Debug, Clone)]
+pub struct Mongo {
+    name: String,
+    tag: String,
+}
+
+impl Default for Mongo {
+    fn default() -> Self {
+        Self {
+            name: NAME.to_owned(),
+            tag: TAG.to_owned(),
+        }
+    }
+}
+
+impl Mongo {
+    pub fn with_name(mut self, name: impl AsRef<str>) -> Self {
+        self.name = name.as_ref().to_string();
+        self
+    }
+
+    pub fn with_tag(mut self, tag: impl AsRef<str>) -> Self {
+        self.tag = tag.as_ref().to_string();
+        self
+    }
```

Comment:
We usually don't expose these methods because it's possible to override with [RunnableImage::with_name](https://docs.rs/testcontainers/latest/testcontainers/core/struct.RunnableImage.html#method.with_name) (and `with_tag`)